Who will participate in the final project?
------------------------------------------

Mariah Taylor and Evan Williams

Will the project will be a data analysis project or a package?
--------------------------------------------------------------

A package that will be used for data analysis. This package will be used to analyze data that we obtain through deep sequencing of viruses and will be used to graphically depict the data.

Which git workflow from the Chacon text you will use?
-----------------------------------------------------

We will use Chacon's recommendation of the Integration-Manager workflow where we each have our own private clone of the project which we will edit different parts of and then we will push our edits together into a finished public repository.

What will the project be about?
-------------------------------

Viruses from the genus of *Hantavirus*are negative sense RNA virus from the family of *Bunyaviridae*. Hantaviruses' genome consisting of 3 segments namely the small segment that encodes for nucleocapsid which encapsulates the genetic material, the medium segment encodes for the envelope glycoproteins which aid in attaching to host cells, and the large segment encodes for the RNA-dependent RNA polymerase.

Single stranded RNA viruses exhibit one of the highest mutation rates found in nature. The high mutation rate is caused by the viruses' RNA polymerase that lack proof reading capabilities and with this introduce mutations or Single/Small Nucleotide Polymorphisms (SNPs) randomly in the virus genome as the virus genome is being replicated. SNPs brought on by the polymerase can either have a beneficial, deleterious or null effect on the virus and its abilities to infect and replicate within the host. SNPs and their effects are what we are currently studying in our laboratory. To study said mutation we perform Next-Gen Sequencing on the virus as they are passaged over time within the host. Data obtained by sequencing is used to determine the number and location of these SNPs across the genomes of these viruses.

The aim of this project will be used to: (1) map the location of the mutations across the genomes along with their frequency at which they occur (2) count the number of SNVs that occur at each passage.

By mapping the location and the frequency of SNPs this will allow researchers to: (1) better understand how viruses mutate within host (2) identify which areas of the virus genome are more conserved and play a more import part in the virus's life which can ultimately be used as possible drug targets.

The package will not only be useful to our laboratory but to any laboratory that currently studying the evolution of viruses which has been a very integral part of virology since the advent of Next-Gen Sequencing.

What is the source of the data? Will real-world data or made-up data be used?
=============================================================================

We will be using a combination of real-world and made up data. We have the data needed for mapping and frequency data analysis, whereas for SNPs count over time we will be using made up data.

What specifications will the final project meet? I.e., what will you produce (a single plot of this vs that, a set of tools that does these three things, etc)
==============================================================================================================================================================

We will produce a single plot for each of the viruses' genome depicting the mapping and frequency of SNVs across each genome. Another plot will be used to depict the number of SNPs observed over time.

For group projects, what are the roles of the group members?
============================================================

Evan will be providing the data for the analysis. Evan will be coding the part of the package that will analyze and depict the mapping of SNPs and their frequency. Mariah will be coding the part of the package that will be used to analyze and depict the number of observed SNPs over time.
